docs/tier1a.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<observations 
+  headers="shortdesc, component, obsDataSet, references"
+  headernames="Dataset, Component, Observational Dataset, References">
+  <aobs>
+    <name>
+      sst Hadley-NOAA
+    </name>
+    <component>
+      ocean
+    </component>
+    <shortdesc>
+      SST global annual climatology and trend
+    </shortdesc>
+    <detaileddesc>
+      Global latitude/longitude plot on ocean native grid. Annual climatologies
+      computed over 5-10 year periods at regular intervals throughout the run.
+      Time series of area-weighted (AW) global mean (1-year running average).
+    </detaileddesc>
+    <obsDataSet>
+      Merged Hadley Center-NOAA/OI data set from Hurrell et al. 2008
+      (https://climatedataguide.ucar.edu/climate-data/merged-hadley-noaaoi-sea-surface-temperature-sea-ice-concentration-hurrell-et-al-2008)
```

Comment:
What if we put this inside `<url>` and `</url>` tags (without the parentheses)?  Then, if the parser sees a `url` tag, it would use the given text with the given URL as a hyperlink.

Reply:
I've done this with Markdown url syntax instead: `[name of url](http://url_link)` as I think this is clearer.